internal/dkg/actions_active.go

```diff
@@ -193,13 +193,10 @@ func (d *Process) StartProposal(
 
 		// migration from v1 -> v2 makes all parties in the v1 group file 'joiners'
 		// the DKGProcess migration path will set old signatures to `nil`, hence we check it here
-	LOOP:
 		for i, j := range currentState.Joining {
 			// leavers may not be online, so we don't want to block the proposal by trying to get their new keys
-			for _, l := range options.Leaving {
-				if j.Address == l.Address {
-					continue LOOP
-				}
+			if util.Contains(options.Leaving, j) {
```

Comment:
I'm not sure this will work in some scenarios - `Signature` will be `nil` before migration, but may not necessarily be so in the proposal

Reply:
Can you confirm / infirm this or should I move back to the original commit?

Reply:
I can confirm this doesn't work for old keys, let's revert it